store/CSDK/src/CarbonReader.h

```diff
@@ -40,6 +40,19 @@ class CarbonReader {
      */
     jobject carbonReaderObject;
 
+    /**
+     * check whether has called builder
```

Comment:
can you improve this comment, suggest you write like "Return true if ...."

Reply:
ok, done